docs/en/query_language/functions/type_conversion_functions.md

```diff
@@ -1,9 +1,69 @@
 
 # Type conversion functions
 
-## toUInt8, toUInt16, toUInt32, toUInt64
+## toInt(8|16|32|64)
```

Comment:
if we decide it's the new way of specifying multiple similar functions, we should change it everywhere separately, instead of doing it during other text changes, otherwise it'll look inconsistent